lib/vector-common/src/internal_event/mod.rs

```diff
@@ -108,7 +110,7 @@ pub struct Count(pub usize);
 
 /// Holds the tuple `(count_of_events, size_of_events_in_bytes)`.
 #[derive(Clone, Copy)]
-pub struct CountByteSize(pub usize, pub usize);
+pub struct CountByteSize(pub usize, pub JsonSize);
```

Comment:
It might just be a naming thing but it strikes me that the doc comment here doesn't mention that the value for the size is from the JSON representation of the events. Does that matter? Just trying to understand if what the struct represents has fundamentally changed and we should call it `CountJsonByteSize` or something 🤷

Reply:
I've updated the comment. Before it was pretty inconsistent as to what byte_size this actually represented. Now it's enforced by the compiler to be the json_size.